### ISSUE TITLE
javascript

### ISSUE THREAD

**Issue by bamioffical** [trelloway/Website-1#2]
Can you add some javascript features and maybe a login panel/ site?

**Comment by trelloway** [comment]
Yeah. I added it to my to do list, i will hit you up when its done. You can watch the progress in projects[https://github.com/trelloway/Website-1/projects/1](url)
